manifests/plugin/container.pp

```diff
@@ -1,6 +1,18 @@
 # @summary Pulp Container plugin
-class pulpcore::plugin::container {
+# @param location_prefix
+#   In the Apache configuration a location with this prefix is exposed. The
+#   version (currently v2) will be appended.
+class pulpcore::plugin::container(
+  String $location_prefix = '/pulpcore_registry',
```

Comment:
I'm not sure there is a reason to make this configurable as far as I can tell. This appears to then make the fragment deviate from the other fragments (https://github.com/theforeman/puppet-pulpcore/pull/105/files#diff-fb6d380303c53d97273a43f661d6d794) as it appears more generic and requires more developer overhead to understand it.

Reply:
Looking at that, I think I can actually generalize the template to redirect additional content URLs. I'm not sure if this needed to be configurable but in my experience having these present as variables is a good practice. I'll take a further look at this.